app/src/main/res/drawable/favorite_project_button.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This button should be the same as the favorite button on the schedule page, so you could call it something more general, i.e. favorite_button.xml